test/system/src/test/java/io/pravega/test/system/BatchClientSimpleTest.java

```diff
@@ -150,7 +151,7 @@ public void batchClientSimpleTest() {
 
         // Instantiate the batch client and assert it provides correct stream info.
         log.debug("Creating batch client.");
-        StreamInfo streamInfo = streamManager.getStreamInfo(SCOPE, stream.getStreamName());
+        StreamInfo streamInfo = streamManager.fetchStreamInfo(SCOPE, stream.getStreamName()).get();
```

Comment:
I think we should use join() method than get(), It will avoid to handle checked exception
we can define method like :
StreamInfo streamInfo = streamManager.fetchStreamInfo(SCOPE, stream.getStreamName()).join();
@tkaitchuck what do you think ?

Reply:
Modified accordingly, used .join() instead of .get(). Thanks @krishan-Rai